bokeh/models/annotations.py

```diff
@@ -439,9 +439,10 @@ class Tooltip(Annotation):
     """
     level = Override(default="overlay")
 
-    side = Either(Auto, Enum(Side), default="auto", help="""
+    attachment = Enum("horizontal", "vertical", "left", "right", "above", "below", help="""
```

Comment:
should there be something like `"auto"` as well? I'm imagining the case (to be handled later) that we might want to automatically dodge multiple tooltips